apps/server/src/modules/school/domain/do/school.ts

```diff
@@ -77,16 +111,12 @@ export interface SchoolProps extends AuthorizableObject {
 	federalState: FederalState;
 	county?: County;
 	purpose?: SchoolPurpose;
-	features: Set<SchoolFeature>;
+	features: SchoolFeature[];
 	systemIds?: EntityId[];
-	logo_dataUrl?: string;
-	logo_name?: string;
+	logo?: SchoolLogo;
 	fileStorageType?: FileStorageType;
 	language?: string;
```

Comment:
In der Response benutzt du `LanguageType` als Typ. Ich glaub, es wäre sinnvoll, den aus user.entity.ts rauszuziehen in shared/domain/types und auch hier einzusetzen.